src/runtime/propensity-server-test.js

```diff
@@ -76,7 +114,7 @@ describes.realWin('PropensityServer', {}, env => {
           return Promise.reject(new Error('Not sent from allowed origin'));
         });
     const eventParam = {'is_active': false, 'offers_shown': ['a', 'b', 'c']};
-    return propensityServer.sendEvent(
+    return propensityServer.sendEvent_(
```

Comment:
this should happen in the then of the callback invocation. first capture the callback from mocking the registerEventListener interface.

Reply:
I think I did what you wanted